assets/sass/main.scss

```diff
@@ -22,48 +22,10 @@ html.wf-active {
 /*
   Styleguide fixes
  */
-
-// Filter modal should cover the screen when opened.
-// Overrides styleguide where the modal is not moved to DOM root.
-.filter-section.modal {
-  .modal-inner {
-    width: 100%;
-    max-width: unset;
-    margin: 0;
-    padding: 0;
-  }
+.sidebar-layout .modal {
+  max-height: unset;
```

Comment:
Are you aware that unset is not supported by IE11...?
https://developer.mozilla.org/en-US/docs/Web/CSS/unset